lang/ui.en.json

```
@@ -767,6 +767,22 @@
     "defaultMessage": "Left arrow icon for MakeCode back",
     "description": "Testing model page title"
   },
+  "makecode-block-default-alt": {
+    "defaultMessage": "ml.onStart(ml.event.{actionName}, function () '{'basic.showIcon(IconNames.{iconName})'}')",
```

I think that's us rather than MakeCode in scenarios where we can't do better.

Just tried MakeCode in VoiceOver and it can be made to read text from the blocks but the DOM order is arbitrary and it is generally awkward.

Perhaps we provide custom alt text here. It's so easy to do in this common/default case.  

(I also don't think it makes sense to translate JavaScript in this way.)

```suggestion
    "defaultMessage": "MakeCode block: on ML {actionName} start, show icon {iconName}",
```

What do you think? Hopefully one day MakeCode will do this for blocks rendering, though without navigation features it would be daunting for large programs.

---

I think it makes sense for this view-only default case [3091a07](https://github.com/microbit-foundation/ml-trainer/pull/434/commits/3091a07211055549850ec7c5910ee21af9f8baae)